tests/jvm/src/test/scala/scala/meta/tests/parsers/BootstrapSuite.scala

```diff
@@ -20,6 +20,7 @@ class BootstrapSuite extends ParseSuite {
       val name = dir.getName
       if (dir.isDirectory && name == "target" || name == "community-projects") return
       def bootstrapTest(src: File): Unit = test("tokenize " + src.getAbsolutePath) {
+        implicit val dialect: Dialect = dialects.Scala33
```

Comment:
why is this needed now?

Reply:
This test would fail while parsing the added scala 3-only files with macros. Alternatively we can filter those out and keep the test as it was.

Reply:
as i mentioned earlier, i'd prefer to have all, or most, tests cover both scala 2 and 3, so their function as regression test will continue to work.

Reply:
was this comment addressed?

Reply:
In the most recent changes, I ended up removing this dialect, and just filtering scala-3 files, preserving previous behavior of the test.